### ISSUE TITLE
how to achieve epoch_callback in allennlp 2.7.0

### ISSUE THREAD

**Issue by rattlesnakey** [allenai/allennlp#5428]
Hello, I use wandb callback in allennlp 2.7.0, and the dashboard on wandb show every batch, I wannna know how to change it into epoch level? also, I wanna know how can I add "learning rate" this hyperparamet into wandb ? thx so much !

**Comment by epwalsh** [comment]
Hi @rattlesnakey, in the W&B dashboard you can chose "Epoch" as the X axis in the graphs:

<img width="443" alt="image" src="https://user-images.githubusercontent.com/8812459/135657020-3cc7d4b2-72f7-49a3-8638-640ecab55b23.png">

> how can I add "learning rate" this hyperparamet into wandb ? 

To log the learning rate, just set `should_log_learning_rate` to `true` in the W&B callback section of your config.

**Comment by rattlesnakey** [comment]
Ok, thx so much!!


| |
hengyuan_blcu
|
|
***@***.***
|
签名由网易邮箱大师定制


On 10/2/2021 ***@***.***> wrote：

Hi @rattlesnakey, in the W&B dashboard you can chose "Epoch" as the X axis in the graphs:

how can I add "learning rate" this hyperparamet into wandb ?

To log the learning rate, just set should_log_learning_rate to true in the W&B callback section of your config.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
